### PR TITLE
Sprint3 updates

### PR DESCRIPTION
**Pull request by technabors**
Font updates, Image handling and page restructuring for all pages within the /layouts folder and other small tweaks. 

JIRAs — RED-1032, 1034, 1037, 1039, 1040

### CODE REVIEW

**Comment by araiford** [comment]
@technabors these look good so far.  Feel free to keep committing work to this branch.  We can just wait until the review this week, and get you to demo the work you've done.  After that we can merge to staging, and then master.

File: _sass/helix/_tables.scss [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+ */
+table {
+  border-collapse: collapse;
+  border-spacing: 0; 
+  border:1px solid #e0e0e0;
+  th {
+  	padding: 1.25em;
+  	border-bottom: 2px solid #0C7C84;
+  	text-align: left;
+  	.1-4 {
```
Comment (araiford) [inline _sass/helix/_tables.scss]: These classes starting with numbers will break.  What effect are you going for with these classes?

File: _includes/website/header.html [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <nav>
-  <div class="logo"><img src="http://c1ee333499ed5f44e56a-fa12562cfe810d69bedcc36a0ac289ef.r55.cf1.rackcdn.com/img/overview/helix-logo-2.svg" class="logo"></div>
+  <div class="logo"><a href="{{site.baseurl}}"><img src="http://c1ee333499ed5f44e56a-fa12562cfe810d69bedcc36a0ac289ef.r55.cf1.rackcdn.com/img/overview/Helix_Logo_%20Final-2.svg" class="logo"></a></div>
```
Comment (araiford) [inline _includes/website/header.html]: adding a / after baseurl will fix the issue of it not working locally:  <a href="{{site.baseurl}}/">

Comment (technabors) [inline _includes/website/header.html]: Issues resolved in commit 2e2c41d on this branch.

File: _sass/helix/_tables.scss (hunk 1 above)
Comment (technabors) [inline _sass/helix/_tables.scss]: Issue resolved in commit 4379bde

**Comment by technabors** [comment]
Can we push this through staging / master branches to close out last sprint?

**Comment by araiford** [comment]
:shipit:

**Comment by technabors** [comment]
Closing branch as this has been merged.

**Pull request merged**